cli/step_list.go

```diff
@@ -1,67 +0,0 @@
-package cli
-
-import (
-	"fmt"
-
-	"github.com/bitrise-io/bitrise/log"
-	"github.com/bitrise-io/bitrise/output"
-	"github.com/bitrise-io/bitrise/tools"
-	"github.com/urfave/cli"
-)
-
-func stepList(c *cli.Context) error {
-	warnings := []string{}
-
-	// Expand cli.Context
-	bitriseConfigBase64Data := c.String(ConfigBase64Key)
-
-	bitriseConfigPath := c.String(ConfigKey)
-
-	format := c.String(OuputFormatKey)
-
-	collectionURI := c.String(CollectionKey)
-	//
-
-	// Input validation
-	if format == "" {
-		format = output.FormatRaw
-	} else if !(format == output.FormatRaw || format == output.FormatJSON) {
-		registerFatal(fmt.Sprintf("Invalid format: %s", format), warnings, output.FormatJSON)
-	}
-
-	if collectionURI == "" {
-		bitriseConfig, warns, err := CreateBitriseConfigFromCLIParams(bitriseConfigBase64Data, bitriseConfigPath)
-		warnings = append(warnings, warns...)
-		if err != nil {
-			registerFatal(fmt.Sprintf("No collection defined and failed to read bitrise config, err: %s", err), warnings, format)
-		}
-
-		if bitriseConfig.DefaultStepLibSource == "" {
-			registerFatal("No collection defined and no default collection found in bitrise config", warnings, format)
-		}
-
-		collectionURI = bitriseConfig.DefaultStepLibSource
-	}
-
-	switch format {
-	case output.FormatRaw:
-		out, err := tools.StepmanRawStepList(collectionURI)
```

Comment:
I'm not removing this because the step **plugin** also imports and calls this function, but it could be a future cleanup